src/librustc/traits/project.rs

```diff
@@ -953,7 +954,28 @@ fn assemble_candidates_from_impls<'cx, 'gcx, 'tcx>(
                         // being invoked).
                         node_item.item.defaultness.has_value()
                     } else {
-                        node_item.item.defaultness.is_default()
+                        let is_default = match selcx.tcx()
```

Comment:
Ok, I'm putting the `TyCtxt`'s method inside [src/librustc/traits/util.rs](https://github.com/rust-lang/rust/blob/master/src/librustc/traits/util.rs#L416) 
the method signature is the following:
`pub fn impl_item_is_default(self, node_item_def_id: DefId) -> bool`